tools/build/bazel_status.sh

```diff
@@ -12,10 +12,6 @@ else
   BRANCH=$(git branch --show-current)
 fi
 
-if [[ $BRANCH == "develop" ]]; then
-  echo STABLE_VERSION "develop"
-else
-  echo STABLE_VERSION $(cat ./VERSION)
-fi
+echo STABLE_VERSION $(cat ./VERSION)
```

Comment:
I removed the version `develop` for the images and the cli, because the `helm charts` cannot have a version `develop`. The version must start with a digit, so we will use the actual version (example: 0.28.0-alpha).